.github/workflows/operatorBuildAndDeploy.yml

```diff
@@ -127,3 +127,42 @@ jobs:
           platforms: linux/amd64
           cache-from: type=local,src=/tmp/.buildx-cache
           cache-to: type=local,dest=/tmp/.buildx-cache
+  push_k8ssandra_update:
```

Comment:
We are at a point now with the projects where we will be releasing new versions of component charts more frequently than the k8ssandra chart. Given that would it make more sense for this to run when a tag is pushed vs every push to master?